ee/codegen/src/generators/graph-attribute.ts

```diff
@@ -316,14 +318,22 @@ export class GraphAttribute extends AstNode {
                 ],
               };
             }
-          } else if (mutableAst.rhs.type === "node_reference") {
-            if (sourceNode && mutableAst.rhs.reference === sourceNode) {
-              return {
-                type: "right_shift",
-                lhs: mutableAst,
-                rhs: { type: "node_reference", reference: targetNode },
-              };
-            }
+            return;
+          }
+
+          const lhsTerminals = getAstTerminals(mutableAst.lhs);
+          const lhsTerminal = lhsTerminals[0];
```

Comment:
Even the sdk itself doesn't support `{A, B} >> {C, D}` yet. We'll need to make sdk-side changes first to do so

Reply:
Is it easy to detect this case and throw an error if we do? My goal is to have code that self-documents this edge case and makes it easy for a future developer to know where to update if/when we do support this.